packages/@lwc/ssr-compiler/package.json

```diff
@@ -47,6 +47,9 @@
             }
         }
     },
+    "imports": {
+        "#estree/*": "./src/estree/*.js"
+    },
```

Comment:
I would avoid package `imports`/`exports`. We've had issues in the past with Rollup/Jest and how it consumes `exports`.